Dockerfile

```diff
@@ -22,6 +22,7 @@ RUN npm install --production \
 # Create actual CryptPad image
 FROM node:lts-slim
 ENV DEBIAN_FRONTEND=noninteractive
+RUN apt update && apt install -y git rdfind && rm -rf /var/lib/apt/lists/*
```

Comment:
There already is an `apt install` line further below. This one can be removed.

Reply:
Yeah, it's gone.